rpcs3/Emu/Audio/AudioBackend.cpp

```diff
@@ -113,9 +113,9 @@ AudioChannelCnt AudioBackend::get_channel_count(u32 device_index)
 	{
 		switch (out.channels)
 		{
-		case 2: return AudioChannelCnt::STEREO;
-		case 6: return AudioChannelCnt::SURROUND_5_1;
-		case 8: return AudioChannelCnt::SURROUND_7_1;
+		case 2: return { AudioChannelCnt::STEREO, AudioChannelCnt::STEREO };
+		case 6: return { AudioChannelCnt::SURROUND_5_1, AudioChannelCnt::SURROUND_5_1 };
+		case 8: return { AudioChannelCnt::SURROUND_7_1, AudioChannelCnt::SURROUND_7_1 };
 		default:
 			fmt::throw_exception("Unsupported channel count in cellAudioOut config: %d", out.channels);
```

Comment:
Similar issue, this will throw an exception now if an invalid channel count was configured.